pyomo/contrib/mindtpy/cut_generation.py

```diff
@@ -110,6 +110,13 @@ def add_ecp_cuts(target_model, solve_data, config,
         constr_vars = list(identify_variables(constr.body))
         jacs = solve_data.jacobians
 
+        if constr.has_lb() and constr.has_ub():
+            config.logger.warning(
+                'constraint {} has both a lower '
+                'and upper bound.'
+                '\n'.format(
+                    constr))
+            break
```

Comment:
Should we terminate when there is an equality constraint? Or skip and go to the next constraint?